map_analysis_2d/ui/control_panels.py

```
@@ -1463,3 +1465,301 @@ def setup_controls(self):
+                label = QLabel(label_text)
+                grid.addWidget(label, row_idx, 0)
+                
+                spins = {}
+                for col_idx, default_val in enumerate([init_val, min_val, max_val], start=1):
+                    spin = QDoubleSpinBox()
+                    spin.setRange(range_min, range_max)
+                    spin.setValue(default_val)
+                    spin.setSingleStep(step)
+                    grid.addWidget(spin, row_idx, col_idx)
+                    
+                    if col_idx == 1: spins['init'] = spin
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)